### DISCUSSION TITLE
ZFSBootMenu systemd Script to make USB Image

### DISCUSSION THREAD

**Discussion by bakerjalexander** [zbm-dev/zfsbootmenu#324] [General]
Working on Ubuntu Bionic
[mkZFSBootMenuImage.sh.txt](https://github.com/zbm-dev/zfsbootmenu/files/9204649/mkZFSBootMenuImage.sh.txt)

**Comment by ofthesun9** [comment]
Thanks for sharing this script, it was helpfull !
